### ISSUE TITLE
cp437 support

### ISSUE THREAD

**Issue by msharov** [chrissimpkins/Hack#83]
Hack can support cp437 (https://en.wikipedia.org/wiki/Code_page_437) with the addition of the following characters:

263A, 263B, 2665, 2666, 2663, 2660, 2642, 2640, 266A, 266B, 263C, 00B7, 207F

While cp437 is not used for writing code, back in DOS days these characters were used to great effect in text-mode games like ZZT and made hex editor output look awesome.

**Comment by txtsd** [comment]
Dwarf Fortress too!

**Comment by chrissimpkins** [comment]
> back in DOS days these characters were used to great effect in text-mode games like ZZT and made hex editor output look awesome

Current use scenarios?

**Comment by msharov** [comment]
No, I am not aware of anyone using it in modern software. But that may well be because so few fonts support it. Extra symbols help make text mode applications look better and cp437 is perhaps the best known set of symbols because it can still be accessed directly through Alt+### codes on Windows. My own interest in this that my current code generates a lot of hex dumps for tracing purposes and I wanted to use cp437 to make them easier to search through. The low values <16 appear most frequently in data and those are the very ones missing from Hack.

**Comment by txtsd** [comment]
> Current use scenarios?

* Dwarf Fortress uses CP437 as its tileset to display all of its default graphics.
* .nfo files use CP437 based ASCII.

**Comment by chrissimpkins** [comment]
Sounds good.  Let me see what I can do.
